src/Shared/runtime/Http3/Helpers/VariableLengthIntegerHelper.cs

```diff
@@ -33,7 +33,7 @@ internal static class VariableLengthIntegerHelper
         private const ulong EightByteLengthMask = 0xC000000000000000;
 
         public const uint OneByteLimit = (1U << 6) - 1;
-        private const uint TwoByteLimit = (1U << 16) - 1;
+        private const uint TwoByteLimit = (1U << 14) - 1;
```

Comment:
Did a new test catch this?

Reply:
Yep. I'll update shared code accordingly.